### ISSUE TITLE
NPE on non-existent entity retrieval.

### ISSUE THREAD

**Issue by Kodehawa** [mewna/catnip#256]
If you try to retrieve an entity (eg. user) that doesn't exist, catnip will greet you with 

```
Caused by: java.lang.NullPointerException
    at com.mewna.catnip.entity.impl.EntityBuilder.createUser(EntityBuilder.java:427)
    at com.mewna.catnip.entity.impl.EntityBuilder.createMember(EntityBuilder.java:636)
    at com.mewna.catnip.rest.handler.RestGuild.lambda$getGuildMember$14(RestGuild.java:409)
    at java.util.concurrent.CompletableFuture.uniApply(CompletableFuture.java:602)
    ... 18 more
```

instead of warning you that the entity doesn't exist.

Discord's response:
```
{
  "code" : 10013,
  "message" : "Unknown User"
}
```

**Comment by Kodehawa** [comment]
(Might be fixed on 0.14.1, needs testing)

**Comment by queer** [comment]
Should be fixed and properly throw a ResponseException as of 0280a6634770647df2cf99ec1b90590be5d680c9
